ssl.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // ssl generates a function to upgrade a net.Conn based on the "sslmode" and
 // related settings. The function is nil when no upgrade should take place.
-func ssl(o values) func(net.Conn) net.Conn {
+func ssl(o values) (f func(net.Conn) (net.Conn, error), err error) {
```

Comment:
What's the reason to use a named return here? I don't see one. Ditto below.

Reply:
Removed.